### ISSUE TITLE
Make Quark accessible for blind visitors

### ISSUE THREAD

**Issue by zebull** [getgrav/grav-theme-quark#123]
I am currently testing the Grav cms with the Quark theme for blind visitors using screen readers. Unfortunately it is not accessible as it is not possible to navigate in the main menu to reach any submenu with the tab key. I have tried to implement the method described on this site:

https://css-tricks.com/solved-with-css-dropdown-menus/

Seemed nice as it is no js. In a custom.css I have tested: 

`.dropmenu ul li a:hover, ul li a:focus-within > ul . . .  `

`.dropmenu.animated ul li a:hover > ul, ul li a:focus-within > ul . . .  `

…in many different variants. By disabling the css in my browser I can make this work. Than it is possible to navigate in (temporarly ugly) Quark dropdown sub-menues with the keyboard and screen reader. With that result in mind I have tried to disable spectre modules one by one. No luck so far, but I will keep on testing the focus-within trick. Any suggestions are very welcome. 

## My feature request: 
Can Quark theme be modified to by default allow dropdown-navigation in common browsers by keyboard tab key? A working example is Wordpress theme Twentyseventeen. 

This inquiry began with my discussion on this forum. 

https://discourse.getgrav.org/t/accessibility-for-blind-visitors/10063?u=zebull

Any help are welcome...

**Comment by NicoHood** [comment]
Did you check https://github.com/getgrav/grav-theme-quark/pull/23 ?
